akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -746,6 +750,15 @@ private final class Sdk(
           context.requestHeaders.allHeaders.asInstanceOf[Seq[HttpHeader]].asJava
 
         override def tracing(): Tracing = new SpanTracingImpl(context.openTelemetrySpan, sdkTracerFactory)
+
+        override def queryParams(): QueryParams = {
+          context.httpRequest.uri.rawQueryString match {
```

Comment:
Use `req.uri.query()` and let the decorator (Scala) implementation work on the Scala query type instead of switch over to Java model for internals.